src/utils/xml_helpers.py

```
@@ -3,30 +3,35 @@
 from typing import Dict, List
```

Ruff (0.8.0)

`typing.Dict` is deprecated, use `dict` instead

(UP035)